docs/en/table_design/indexes/Bitmap_index.md

```diff
@@ -29,7 +29,11 @@ You can benefit from bitmap indexes in the following aspects:
 
 - You can create a bitmap index for a column that can be filtered by using the equal (`=`) or [NOT] IN operator.
 - You can create bitmap indexes for all columns of a Duplicate Key table or Unique Key table. For an Aggregate table or Primary Key table, you can only create bitmap indexes for key columns.
-- The columns of the FLOAT, DOUBLE, BOOLEAN, and DECIMAL types do not support creating bitmap indexes.
+- Bitmap indexes can be created for the columns of the following  types:
```

Comment:
![image](https://github.com/StarRocks/starrocks/assets/57167462/36c3beb3-2d20-45e4-a4c3-4819e6b2a784)
1. remove "the"
2. extra space
3. types -> data types